### DISCUSSION TITLE
Is there a manuscript to reference yet?

### DISCUSSION THREAD

**Discussion by rwbaer** [DillonHammill/CytoExploreR#175] [Q&A] [answered]
I thhink I recall you working on a manuscript that describes cytoExploreR some time back.  Did that every get completed, and if so how might I reference it. 

 If not, is   cytation("cytoExploreR") still our best bet?
Hammill D (2021). _CytoExploreR: Interactive Analysis of Cytometry Data_. R package version 1.1.0,
  <https://github.com/DillonHammill/CytoExploreR>.

**Comment by DillonHammill** [comment, marked as answer]
Yes, a manuscript for version 2.0.0 is in the works and you will be able to cite that soon.

If you are using version 1.1.0 it is best to cute it as indicated in the README.

**Reply by rwbaer** [reply]
Thanks

**Comment by rwbaer** [comment]
As I have still not seen an announcement for 2.0.0 beta, I'm still using 1.1.0.  The citation in that version produces the following in quarto reference:

```
Hammill, Dillon. 2021. “CytoExploreR: Interactive Analysis of Cytometry Data.” https://github.com/DillonHammill/CytoExploreR.
```

This takes you straight to Github.  I'd like to encourage you to change this to your very helpful set of startup guides. You already provide a link straight to github from there.  Perhaps citation() should yileld

```
Hammill, Dillon. 2021. “CytoExploreR: Interactive Analysis of Cytometry Data.” https://dillonhammill.github.io/CytoExploreR/.
```
until the 2.0.0 paper is ready.
